.travis.yml

```diff
@@ -7,6 +7,7 @@ python:
 # command to install dependencies
 # some are only used for travis/coveralls so we are installing them here only
 install:
+  - sudo useradd -p $1$/goz4USO$3QNU7YVvUbgyhAuROfTAw1 alignak
```

Comment:
we shoud try to remove all sudo actually to migrate to the new travis platform (much faster) . If you don't require alignak user to install alignak this is not necessary